### DISCUSSION TITLE
当binlog-format=STATEMENT用于我的MySQL实例时，应该如何配置archery

### DISCUSSION THREAD

**Discussion by anyuqingcloud** [hhyo/Archery#2158] [Q&A]
当提交关于archery的SQL审计时，archery将首先执行set session-binlog format=ROW，因为我的MySQL实例binlog-format=STATEMENT，如果设置为ROW，将对主从同步产生影响，并导致其断开连接 ,该怎么解决

**Comment by nick2wang** [comment]
goinception备份的必要条件是format=ROW，如果你不需要，取消备份即可

**Reply by anyuqingcloud** [reply]
在哪进行取消啊

**Reply by nick2wang** [reply]
> 在哪进行取消啊

![image](https://github.com/hhyo/Archery/assets/33473924/1f1489d2-fe8a-4413-a855-151eee74d5b6)

**Reply by anyuqingcloud** [reply]
这个默认就是关闭的，需要打开吗

**Reply by nick2wang** [reply]
> 这个默认就是关闭的，需要打开吗

打开之后才可以在页面上选择不备份

**Reply by anyuqingcloud** [reply]
ok了，非常感谢大哥
